lms/static/js/instructor_dashboard/membership.js

```diff
@@ -463,11 +463,26 @@ such that the value can be defined later than this assignment (file load order).
             if (successes.length && dataFromServer.action === 'add') {
                 // Translators: A list of users appears after this sentence;
                 renderList(gettext('These users were successfully added as beta testers:'), (function() {
-                    var j, len1, results;
+                    var j, k, len1, results, inActive, message, inActiveUsers;
                     results = [];
+                    inActiveUsers = [];
+                    inActive = false;
                     for (j = 0, len1 = successes.length; j < len1; j++) {
                         sr = successes[j];
-                        results.push(sr.identifier);
+                        if (sr.is_active) {
+                            results.push(sr.identifier);
+                        } else {
+                            inActiveUsers.push(sr.identifier);
+                            inActive = true;
+                        }
+                    }
+                    if (inActive) {
```

Comment:
instead of maintaining a separate `inActive` boolean, can you just check that `inActiveUsers` isn't empty? That simplifies the code a little